plasma/cli/client.py

```diff
@@ -0,0 +1,128 @@
+import click
+import plyvel
+import rlp
+from ethereum import utils
+from .main import main
+from plasma.utils.utils import confirm_tx
+from plasma.client.client import Client
+from plasma.child_chain.child_chain import Block
+from plasma.child_chain.transaction import Transaction
+
+
+@main.command('start')
```

Comment:
Possibly not a great UX. `omg start <command>` is more annoying to use than `omg <command>`.